app/controllers/annotations_controller.rb

```diff
@@ -1,9 +1,28 @@
 class AnnotationsController < ApplicationController
-  before_filter :authenticate_user!
+    before_filter :authenticate_user!
 
-  def index
-  end
+    def index
+        loadOptions = {
+            :limit =>       100,
+            :groups =>      current_user.rep_group_list,
+            :subgroups =>   current_user.rep_subgroup_list,
+            :host =>        request.host_with_port,
+            :user =>        current_user.email,
+            :context =>     'search'
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.